gum/backend-qnx/gumprocess-qnx.c

```diff
@@ -55,6 +60,14 @@ struct _GumFindExportContext
   const gchar * symbol_name;
 };
 
+struct _dl_phdr_internal {
```

Comment:
Structs should always be following the camel-case style of `GumFooBar`, so here it could be something along the lines of `GumDlPhdrInternal`.